lib/util/git.js

```diff
@@ -72,6 +129,38 @@ exports.clone = function(repoUrl, repoPath) {
   }).return();
 };
 
+exports.commit = function(repoPath, files, message) {
```

Comment:
really wish we were using node-git here.  @tbranyen, could we use node-git for any of this?